### PR TITLE
Heirloom Expansion

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Adds unique items, and new items for the heirloom trait.

## Why It's Good For The Game

Makes sense lore-wise. Who the fuck cherishes a bag of dice/pack of cards from their parents?

## Changelog
:cl:
tweak: Family heirloom trait now offers more items 
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
>Who the fuck cherishes a bag of dice/pack of cards from their parents?

You have no fucking idea.

**Comment by ghost** [comment]
why THE FUCK do felinids have a cattoy heirloom

**Comment by eeSPee** [comment]
Probably gonna atomize this,

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -120,17 +122,23 @@
 				heirloom_type = /obj/item/storage/toolbox/mechanical/old/heirloom
 			if("Barber")
 				heirloom_type = /obj/item/handmirror
+			if("Psychiatrist")
+				heirloom_type =  pick(subtypesof(/obj/item/toy/plush))
 			if("Stage Magician")
 				heirloom_type = /obj/item/gun/magic/wand
+			if("VIP")
+				heirloom_type = /obj/item/pda/clear
```
Comment (zeskorion) [inline code/datums/traits/negative.dm:130]: no use in two pdas

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -95,15 +95,17 @@
...
+	else if(is_species(H, /datum/species/felinid) && prob(10))
+		heirloom_type = /obj/item/toy/cattoy
+	else if (prob(90))
 		switch(quirk_holder.mind.assigned_role)
 			//Service jobs
 			if("Clown")
 				heirloom_type = /obj/item/bikehorn/golden
 			if("Mime")
-				heirloom_type = /obj/item/reagent_containers/food/snacks/baguette
+				heirloom_type = /obj/item/staff/stick
```
Comment (zeskorion) [inline code/datums/traits/negative.dm:108]: arent these too big to fit a backpack?

File: code/datums/traits/negative.dm [hunk 3]
```diff
@@ -174,9 +182,10 @@
 
 	if(!heirloom_type)
 		heirloom_type = pick(
-		/obj/item/toy/cards/deck,
-		/obj/item/lighter,
-		/obj/item/dice/d20)
+		/obj/item/reagent_containers/food/drinks/urn,
```
Comment (zeskorion) [inline code/datums/traits/negative.dm:185]: this means the datum is tied to the urn, not the ashes- drinking the ashes has no negative effect

File: code/modules/clothing/under/accessories.dm [hunk 4]
```diff
@@ -367,4 +367,24 @@
...
 	new /obj/item/ammo_box/c38(src)
-	new /obj/item/ammo_box/c38(src)
\ No newline at end of file
+	new /obj/item/ammo_box/c38(src)
+
+///////////////////
+//FAMILY HEIRLOOM//
+///////////////////
+
+/obj/item/clothing/accessory/heirloom
```
Comment (zeskorion) [inline code/modules/clothing/under/accessories.dm:376]: copypasted lawyer badge code, without even knowing what it does

File: code/datums/traits/negative.dm [hunk 5]
```diff
@@ -120,17 +122,23 @@
 				heirloom_type = /obj/item/storage/toolbox/mechanical/old/heirloom
 			if("Barber")
 				heirloom_type = /obj/item/handmirror
+			if("Psychiatrist")
+				heirloom_type =  pick(subtypesof(/obj/item/toy/plush))
```
Comment (zeskorion) [inline code/datums/traits/negative.dm:126]: they should not get normal narsie or ratvar plushies

File: code/datums/traits/negative.dm [hunk 6]
```diff
@@ -120,17 +122,23 @@
...
 			if("Stage Magician")
 				heirloom_type = /obj/item/gun/magic/wand
+			if("VIP")
+				heirloom_type = /obj/item/pda/clear				
 			//Security/Command
 			if("Captain")
-				heirloom_type = /obj/item/reagent_containers/food/drinks/flask/gold
+				heirloom_type = pick(/obj/item/reagent_containers/food/drinks/flask/gold, /obj/item/phone)
```
Comment (zeskorion) [inline code/datums/traits/negative.dm:133]: why the phones in all of these?

File: code/datums/traits/negative.dm [hunk 7]
```diff
@@ -120,17 +122,23 @@
...
+			if("Head Of Personnel")
+				heirloom_type = pick(/obj/item/toy/figure/ian, /obj/item/phone)
 			if("Head of Security")
-				heirloom_type = /obj/item/book/manual/wiki/security_space_law
+				heirloom_type = pick(/obj/item/book/manual/wiki/security_space_law, /obj/item/phone)
 			if("Warden")
 				heirloom_type = /obj/item/book/manual/wiki/security_space_law
 			if("Security Officer")
-				heirloom_type = pick(/obj/item/book/manual/wiki/security_space_law, /obj/item/clothing/head/beret/sec)
+				heirloom_type = pick(/obj/item/clothing/head/beret/sec, /obj/item/clothing/head/helmet/old)
```
Comment (zeskorion) [inline code/datums/traits/negative.dm:141]: old helmet is worse than useless. just remove the spacelaw book

File: code/datums/traits/negative.dm [hunk 8]
```diff
@@ -139,7 +147,7 @@
 				heirloom_type = pick(/obj/item/clothing/neck/stethoscope, /obj/item/book/manual/wiki/security_space_law)
 			//RnD
 			if("Research Director")
-				heirloom_type = /obj/item/toy/plush/slimeplushie
+				heirloom_type = pick(/obj/item/toy/plush/slimeplushie, /obj/item/phone)
```
Comment (zeskorion) [inline code/datums/traits/negative.dm:150]: WHY THE PHONES

**Review by zeskorion — CHANGES_REQUESTED** [review]

File: code/datums/traits/negative.dm (hunk 7 above)
Comment (PowerfulBacon) [reply code/datums/traits/negative.dm:141]: Isn't that the point of heirlooms, to be a burden?

File: code/datums/traits/negative.dm (hunk 8 above)
Comment (eeSPee) [reply code/datums/traits/negative.dm:150]: They're clunky, cannot be worn and are useless. As a family heirloom should be?

**Comment by eeSPee** [comment]
I think I will atomize this into two PRs;
One to change the mime's heirlooms
Another to change the base heirlooms into more lorefriendly items

**Comment by eeSPee** [comment]
I'll have to think this through.

**Pull request closed without merging**